src/main/java/com/ipcounter/classes/CounterStorage.java

```diff
@@ -0,0 +1,58 @@
+package com.ipcounter.classes;
```

Comment:
package with name "classes" is very general. here you can put all classes of application. It is not main goal of packaging. Packaging gives us possibility to group classes by purpose.